src/absil/il.fs

```diff
@@ -18,6 +18,7 @@ open System.Collections
  
 let logging = false 
 
+#if FX_RUNNING_ON_MONO
```

Comment:
Ok, I see.  In any case my comment was more about minimizing the diff - for about half the cases the runningOnMono code does compile but will not be activated if runningOnMOno were false